SQL/0000-00-03-ConfigTables.sql

```diff
@@ -17,7 +17,7 @@ CREATE TABLE `ConfigSettings` (
     `Description` varchar(255) DEFAULT NULL,
     `Visible` tinyint(1) DEFAULT '0',
     `AllowMultiple` tinyint(1) DEFAULT '0',
-    `DataType` ENUM('text', 'boolean', 'email', 'instrument', 'textarea', 'scan_type', 'lookup_center') DEFAULT NULL,
+    `DataType` ENUM('text', 'boolean', 'email', 'instrument', 'textarea', 'scan_type', 'lookup_center', 'path', 'web_path') DEFAULT NULL,
```

Comment:
I think we need an imaging_path type to differentiate imaging pipeline paths from other paths. I'm also not crazy about the term "web_path" because it sounds like it's refering to the path part of a URL, but I'm not sure what a better name would be.

Reply:
maybe like `writable_path` or something?

Reply:
Are the imaging paths limited to: 
'mincPath', 'DownloadPath', 'IncomingPath', 'MRICodePath', 'MRIUploadIncomingPath', 'imagePath'
?

I haven't done much work with the imaging pipeline so I'm not sure.